openwisp_radius/base/models.py

```diff
@@ -318,6 +324,11 @@ class Meta:
     def __str__(self):
         return self.username
 
+    def clean(self):
+        super().clean()
+        if self.user and not self.user.is_member(self.organization):
+            raise ValidationError({'organization': _ORGANIZATION_HELP_TEXT})
```

Comment:
we already have logic to deal with these cases in the rest of openwisp, take a look at openwisp-users and openwisp-controller.

Reply:
Take a look at these methods: https://github.com/openwisp/openwisp-users/blob/master/openwisp_users/mixins.py
Then look in openwisp-controller for these to see how they're used.

Reply:
@nemesisdesign I have tried using the [_validate_org_relation](https://github.com/openwisp/openwisp-users/blob/master/openwisp_users/mixins.py#L16) to check if the selected user is in the same organization as the  RadiusReply but I get the error, `'User' object has no attribute 'organization_id'`. 
Looking at the implementation of `_validate_org_relation`, it seems the relating object must have an `organization_id` field.

This is my implementation in models.py
```
    def clean(self):
        super().clean()
        if self.user:
            self._validate_org_relation('user') 
```

Reply:
ok, you're right, please just avoid the duplication of this method, it's not necessary to define it in `AbstractRadiusReply` since this class inherits `AutoUsernameMixin` which has the same code. Moreover, please always avoid duplicating any code. Follow the DRY principle.

Reply:
Okay, thanks for the feedback.